image-trigger

```diff
@@ -69,8 +69,9 @@ REFRESH = {
 
 
 def main():
-    parser = argparse.ArgumentParser(description="Ensure necessary issue exists for image refresh")
-    parser.add_argument("-v", "--verbose", action="store_true", default=False, help="Print verbose information")
+    parser = argparse.ArgumentParser(description='Ensure necessary issue exists for image refresh')
```

Comment:
This "rebase" commit is weird and confusing -- please do `git fetch origin; git rebase -i origin/main` and eliminate it, then force-push it here. That will ensure a clean set of "real" commits to review.

Reply:
See [here](https://github.com/cockpit-project/bots/actions/runs/8360338384/job/22886233816?pr=5954), this makes rebasing fail.

Reply:
done!